### PR TITLE
Update LATAM channels

### PR DESCRIPTION
**Pull request by Carlinhos027**

### CODE REVIEW

**Comment by BellezaEmporium** [comment]
I might have a few LATAM channels we may add to the playlists. I'll simply need to reverify the servers that are in https://github.com/orgs/iptv-org/discussions/213

File: streams/ec.m3u [hunk 1]
```diff
@@ -35,24 +35,44 @@ https://stmv1.voxplayer.com.br/vidieg12/vidieg12/playlist.m3u8
...
+#EXTINF:-1 tvg-id="MetropoliMediosTV.ec",Metropoli Medios TV (720p)
+https://eu1.servers10.com:8081/8214/index.m3u8
 #EXTINF:-1 tvg-id="MulticanalCatamayo.ec",Multicanal Catamayo (720p) [Not 24/7]
 https://multicanal.streamseguro.com/hls/streaming.m3u8
 #EXTINF:-1 tvg-id="Nortvision.ec",Nortvisión (720p)
 https://plataformavideo.com/live/nortvision.m3u8
+#EXTINF:-1 tvg-id="NukaTV.ec",Ñuka TV (720p)
+https://cloudvideo.servers10.com:8081/8118/index.m3u8
+#EXTINF:-1 tvg-id="OromarTV.ec",Oromar TV (720p)
+https://stream.oromartv.com:8082/hls/oromartv_hi/index.m3u8
```
Comment (BellezaEmporium) [inline streams/ec.m3u:51]: https://stream.oromartv.com:8082/hls/oromartv.m3u8 ?

the "low" quality doesn't load, the rest does.

Comment (Carlinhos027) [reply streams/ec.m3u:51]: I know, that's kinda weird lol

**Comment by sguinetti** [comment]
Hello. I update new streams from SEO TV. Now, the network has nine channels.

File: streams/co.m3u [hunk 2]
```diff
@@ -9,8 +11,12 @@ https://movil.ejeserver.com/live/verteve.m3u8
 https://video.ejeserver.com/live/verteve.m3u8
 #EXTINF:-1 tvg-id="AvivamientoTV.co",Avivamiento TV (1080p) [Not 24/7]
 https://s1.abntelevision.com/avivamientoabr/stream/avivamientohd/avivamientohd/playlist.m3u8
+#EXTINF:-1 tvg-id="AzufralCanalRegional.co",Azufral Canal Regional (720p)
+https://tvlatina.live:1936/8020/8020/playlist.m3u8
```
Comment (Dum4G) [inline streams/co.m3u:15]: 404

File: streams/co.m3u [hunk 3]
```diff
@@ -54,27 +60,33 @@ https://movil.ejeserver.com/live/visiondorada.m3u8
...
 https://mdstrm.com/live-stream-playlist/574463697b9817cf0886fc17.m3u8
+#EXTINF:-1 tvg-id="CarareTV.co",Carare TV (1080p) [Not 24/7]
+https://cp.panelchs.com:1936/8160/8160/playlist.m3u8
 #EXTINF:-1 tvg-id="CinemaPlus.co",Cinema + (720p)
 https://byecableiptvnew3.ddns.net/ENVIVOCINEMA/index.m3u8
-#EXTINF:-1 tvg-id="CNCHDOnline.co",CNC HD Online (1080p)
+#EXTINF:-1 tvg-id="CitytvBogota.co",Citytv Bogotá (1080p) [Not 24/7]
+https://ythls.armelin.one/channel/UCv5Oy_TJFvGKXjeNzzJE_PQ.m3u8
+#EXTINF:-1 tvg-id="CiudadelaTV.co",CiudadelaTV (614p)
+https://cp.panelchs.com:1936/8096/8096/playlist.m3u8
```
Comment (Dum4G) [inline streams/co.m3u:70]: 404

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: streams/co.m3u (hunk 2 above)
Comment (BellezaEmporium) [reply streams/co.m3u:15]: Not 24/7

File: streams/co.m3u (hunk 3 above)
Comment (BellezaEmporium) [reply streams/co.m3u:70]: Not 24/7

Comment (Carlinhos027) [reply streams/co.m3u:70]: They are not 24/7.

File: streams/co.m3u [hunk 4]
```diff
@@ -9,8 +11,12 @@ https://movil.ejeserver.com/live/verteve.m3u8
 https://video.ejeserver.com/live/verteve.m3u8
 #EXTINF:-1 tvg-id="AvivamientoTV.co",Avivamiento TV (1080p) [Not 24/7]
 https://s1.abntelevision.com/avivamientoabr/stream/avivamientohd/avivamientohd/playlist.m3u8
+#EXTINF:-1 tvg-id="AzufralCanalRegional.co",Azufral Canal Regional (720p)
```
Comment (Dum4G) [inline streams/co.m3u:14]:
```suggestion
#EXTINF:-1 tvg-id="AzufralCanalRegional.co",Azufral Canal Regional (720p) [Not 24/7]
```

File: streams/co.m3u [hunk 5]
```diff
@@ -54,27 +60,33 @@ https://movil.ejeserver.com/live/visiondorada.m3u8
...
 #EXTINF:-1 tvg-id="CaracolTV.co",Caracol TV (720p) [Geo-blocked]
 https://mdstrm.com/live-stream-playlist/574463697b9817cf0886fc17.m3u8
+#EXTINF:-1 tvg-id="CarareTV.co",Carare TV (1080p) [Not 24/7]
+https://cp.panelchs.com:1936/8160/8160/playlist.m3u8
 #EXTINF:-1 tvg-id="CinemaPlus.co",Cinema + (720p)
 https://byecableiptvnew3.ddns.net/ENVIVOCINEMA/index.m3u8
-#EXTINF:-1 tvg-id="CNCHDOnline.co",CNC HD Online (1080p)
+#EXTINF:-1 tvg-id="CitytvBogota.co",Citytv Bogotá (1080p) [Not 24/7]
+https://ythls.armelin.one/channel/UCv5Oy_TJFvGKXjeNzzJE_PQ.m3u8
+#EXTINF:-1 tvg-id="CiudadelaTV.co",CiudadelaTV (614p)
```
Comment (Dum4G) [inline streams/co.m3u:69]:
```suggestion
#EXTINF:-1 tvg-id="CiudadelaTV.co",CiudadelaTV (614p) [Not 24/7]
```

**Review by Dum4G — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**